projects/cds/ng-package.json

```diff
@@ -0,0 +1,12 @@
+{
+  "$schema": "../../node_modules/ng-packagr/ng-package.schema.json",
+  "dest": "../../dist/cds",
+  "lib": {
+    "entryFile": "./public_api.ts",
+    "umdModuleIds": {
+      "@spartacus/core": "core",
+      "rxjs": "rxjs",
+      "rxjs/operators": "operators"
```

Comment:
@znikola we weren't 100% sure exactly which external dependencies we should be recording here. Some of the other libraries seemed to be including some ngrx things, but not other external dependencies. Should we just be including `@spartacus/core` here for now?